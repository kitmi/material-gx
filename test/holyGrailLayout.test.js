import React from 'react';
import renderer from 'react-test-renderer';
import { holyGrailLayoutCreator } from '../build';

const HolyGrailLayout = holyGrailLayoutCreator(240);

it('renders correctly', () => {
    const tree = renderer
        .create(
            <HolyGrailLayout
                header={<div>header</div>}
                left={<div>left sidebar</div>}
                right={<div>right sidebar</div>}
                content={<div>content</div>}
                footer={<div>footer</div>}
            />
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
