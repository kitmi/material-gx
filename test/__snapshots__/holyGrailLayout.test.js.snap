// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="HolyGrail-root-1"
>
  <div
    className="HolyGrail-headerFooter-2"
  >
    <div>
      header
    </div>
  </div>
  <div
    className="HolyGrail-body-3"
  >
    <div
      className="HolyGrail-content-4"
    >
      <div>
        content
      </div>
    </div>
    <div
      className="HolyGrail-left-5 HolyGrail-side-6"
    >
      <div>
        left sidebar
      </div>
    </div>
    <div
      className="HolyGrail-side-6"
    >
      <div>
        right sidebar
      </div>
    </div>
  </div>
  <div
    className="HolyGrail-headerFooter-2"
  >
    <div>
      footer
    </div>
  </div>
</div>
`;
